src/doctrines/Preorders.jl

```diff
@@ -0,0 +1,60 @@
+export Preorder, ThinCategory, MonoidalThinCategory, FreePreorder,
+  FreeThinCategory, FreeMononoidalThinCategory
+
+import Catlab.Doctrines: Ob, Hom, otimes, compose, ⋅, ⊗
```

Comment:
For once we don't need these imports, because all the doctrine files get included into a single module.